climakitae/core/data_export.py

```diff
@@ -199,17 +229,50 @@ def _export_to_netcdf(data, save_name):
         data to export to NetCDF format
     save_name : string
         desired output file name, including the file extension
+    mode : string
+        localtion logic for storing export file.
 
     Returns
     -------
     None
 
     """
-    est_file_size = _estimate_file_size(data, "NetCDF")
-    disk_space = shutil.disk_usage("./")[2] / bytes_per_gigabyte
+    print("Exporting specified data to NetCDF...")
+
+    # Convert xr.DataArray to xr.Dataset so that compression can be utilized
+    _data = data
+    if isinstance(_data, xr.core.dataarray.DataArray):
+        if not _data.name:
+            # name it in order to call to_dataset on it
+            _data.name = "data"
+        _data = _data.to_dataset()
+
+    est_file_size = _estimate_file_size(_data, "NetCDF")
+    disk_space = shutil.disk_usage(os.path.expanduser("~"))[2] / bytes_per_gigabyte
+
+    _warn_large_export(est_file_size)
+    _update_attributes(_data)
+    _update_encoding(_data)
+
+    file_location = "local"
+
+    if mode == "local":
+        if disk_space <= est_file_size:
+            raise Exception("Data too large to save locally. Use the mode=s3 option.")
```

Comment:
“locally” might be confused with the “local machine” phrase printed after a save succeeds. “s3” can be confusing too, especially to non-technical users.

Reply:
Very valid point. I will come up with alternative language. Thx

Reply:
I've decided to keep the language the same and just explain it more